schedule/containers/sle_image_on_sle_ecr.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: sle_image_on_publiccloud
```

Comment:
```suggestion
name: sle_image_on_ecr
```

Reply:
File name `sle_image_on_amazon_ecr` better? Now it seems `ecr` is something from `sle` which is wrong.

Reply:
ok, thanks

Reply:
yeah I also have concerns regarding this . even more in all other cases naming convention is `<container OS>_on_<Host OS>`  so `amazon ecr` is better than `sle ecr ` but still it is not OS . I would actually go with approach which we have for uploading "normal" images . where we have `upload_image` scenario which based on variables provided call appropriate provider function for upload. So in this case it will be `upload_container_image`  . WDYT ?

Reply:
done, I changed by sle_image_on_amazon_ecr that includes the name of amazon